app/views/dashboards/show.html.erb

```diff
@@ -5,27 +5,28 @@
   <%= link_to "Save a snippet", new_snippet_path %>
   <%= link_to "Save an audio file", new_audio_file_path %>
   <%= link_to "Save a wishlist item", new_wishlist_path %>
-  <h2>🎹 Latest music snippets: 🎸</h2>
+  <h2>🎹 Latest music snippet activity: 🎸</h2>
 </div>
 
+<div class="snippet">
+  <% @activities.each_with_index do |activity, i| %>
+    <h3 class="snippet-info"><%= activity.subject.title %>, by <%= activity.subject.artist %></h3>
+    <p class="saved-by"><%= activity.subject_type.underscore.humanize %> saved by: <%= activity.user.username %></p>
+    <%= render activity.subject_type.underscore, activity: activity %>
```

Comment:
Consider utilizing `to_partial_path` here so that this becomes:

```erb
<%= render activity.subject %>
```

This will involve moving the around the partials you created for `dashboards/_snippet` etc.. and changing their contents a little but simplifying their contents.